packages/flutter_sprites/README.md

```diff
@@ -1,8 +1,209 @@
+    
+    class MyWidget extends StatefulWidget {
+      @override
+      MyWidgetState createState() => new MyWidgetState();
+    }
+    
+    class MyWidgetState extends State<MyWidget> {
+      NodeWithSize rootNode;
+      
+      @override
+      void initState() {
+        super.initState();
+        rootNode = new NodeWithSize(const Size(1024.0, 1024.0));
+      }
+      
+      @override
+      Widget build(BuildContext context) {
+      	return new SpriteWidget(rootNode);
+      }
+    }
+
+By default the SpriteWidget uses letterboxing to display its contents. This means that the size that you give the root node will determine how the SpriteWidget's contents will be scaled to fit. If it doesn't fit perfectly in the area of the widget, either its top and bottom or the left and right side will be trimmed. You can optionally pass in a parameter to the SpriteWidget for other scaling options depending on your needs.
+
+## Adding objects to your node graph
+Your SpriteWidget manages a node graph, the root node is the NodeWithSize that is passed in to the SpriteWidget when it's created. To render sprites, particles systems, or any other objects simply add them to the node graph.
+
+Each node in the node graph has a transform. The transform is inherited by its children, this makes it possible to build more complex structures by grouping objects together as children to a node and then manipulating the parent node. For example the following code creates a car sprite with two wheels attached to it. The car is added to the root node.
+
+    Sprite car = new Sprite.fromImage(carImage);
+    Sprite frontWheel = new Sprite.fromImage(wheelImage);
+    Sprite rearWheel = new Sprite.fromImage(wheelImage);
+    
+    frontWheel.position = const Point(100, 50);
+    rearWheel.position = const Point(-100, 50);
+    
+    car.addChild(frontWheel);
+    car.addChild(rearWheel);
+    
+    rootNode.addChild(car);
+    
+You can manipulate the transform by setting the position, rotation, scale, and skew properties.
+
+## Sprites, textures, and sprite sheets
+The most common node type is the Sprite node. A sprite simply draws an image to the screen. Sprites can be drawn from Image objects or Texture objects. A texture is a part of an Image. Using a SpriteSheet you can pack several texture elements within a single image. This saves space in the device's gpu memory and also make drawing faster. Currently Flutter Sprites supports sprite sheets in json format and produced with a tool such as TexturePacker. You can create a SpriteSheet with a definition in json and an image:
+
+    SpriteSheet sprites = new SpriteSheet(myImage, jsonCode);
+    Texture texture = sprites['texture.png'];
+
+## The frame cycle
+Each time a new frame is rendered to screen Flutter Sprites will perform a number of actions. Sometimes when creating more advanced interactive animations or games, the order in which these actions are performed may matter.
+
+This is the order things will happen:
```

Comment:
Please list the phases using an ordered (numbered) list rather than as bullet points. Also, please reposition the subsequent sections so that they match the order that you've provided ('Handling user input' section should come first, followed by 'Animating using actions', then 'Constraints' etc.).